docs/source/alignment.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>I did this in a oneliner, probably in the first PR/issue discussing this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Yeah but the one liner breaks frequently in the notebooks. You have to keep restarting it for it to work. And once you set it a certain way you have to restart to change it. This solution allows you to modify it as you go. Since  <code>logging</code>  is a standard library, they can use their own strategy.</p>